README.md

```diff
@@ -16,68 +16,75 @@ To get started with the demo app, please ensure Flutter is installed on your sys
     // Other dependencies...
     implementation 'com.example.adapter:version'
 }
 ```
+> [!CAUTION]
+> Do not add the underlying AppLovin SDK to your dependencies. The AppLovin MAX Flutter plugin
+> already specifies the certified SDK version corresponding to the plugin. Manually adding it may
+> break your build and cause unpredictable results.
+>
+> ~~implementation 'com.applovin:applovin-sdk:+'~~
 
-#### 2. Updating `AndroidManifest.xml` with `meta-data` (if required):
-- Within the `android/app/` directory, locate and open the `AndroidManifest.xml` file. 
-- If the network adapter you add requires `meta-data`, insert the necessary `meta-data` elements within the `<application>` tag. 
-
-#### 3. Adding Java or Kotlin Code (if required):
-   
-For Java: 
-- Navigate to `android/app/src/main/java/com/example/your_project_name`
-- Open `MainActivity.java` with a text editor or IDE. 
-- Add the required Java code to this file, usually within the `configureFlutterEngine` method. 
-```
-@Override
-public void configureFlutterEngine(FlutterEngine flutterEngine) {
-    super.configureFlutterEngine(flutterEngine);
-    // Your Java code here...
-}
-```
-
-For Kotlin: 
-- Navigate to `android/app/src/main/kotlin/com/example/your_project_name`
-- Open `MainActivity.kt` with a text editor or IDE. 
-- Add the required Kotlin code to this file to this file, usually within the `configureFlutterEngine` method. 
-```
-override fun configureFlutterEngine(flutterEngine: FlutterEngine) {
-    super.configureFlutterEngine(flutterEngine)
-    // Your Kotlin code here...
-}
-```
+#### 3. Updating `AndroidManifest.xml` with `meta-data` (if required):
+- Within the `android/app/` directory, locate and open the `AndroidManifest.xml` file.
+- If the mediated ad network adapter you add requires `meta-data`, insert the necessary `meta-data` elements within the `<application>` tag. 
 
 ### iOS 
 #### 1. Adding adapters to your podfile:
 - Locate your `Podfile` in the `/ios` folder.
-- Open the Podfile with a text editor or IDE.
-- Add the adapter pods to your application as specified in the [documentation](https://developers.applovin.com/en/flutter/preparing-mediated-networks#ios). It will look something like this
+- Open the `Podfile` with a text editor or IDE.
+- Add the necessary adapter pods for the mediated networks you're integrating, as specified in the [documentation](https://developers.applovin.com/en/flutter/preparing-mediated-networks#ios). It will look something like this
 ```
 target 'Runner' do
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
-  pod `AppLovinSDK`
+  pod 'AppLovinMediationExampleAdapter'
 end
 ```
+
+> [!CAUTION]
+> Do not add the underlying AppLovin SDK to your dependencies. The AppLovin MAX Flutter plugin
+> already specifies the certified SDK version corresponding to the plugin. Manually adding it may
+> break your build and cause unpredictable results.
+>
+> ~~pod 'AppLovinSDK'~~
+
 #### 2. Install the Pods:
```

Comment:
For better parallelism with the rest of the headings on this page, change this to "2. Install**ing** the pods:"